tests/test_parsing.py

```diff
@@ -179,6 +181,10 @@ def test_find_imports_from_text__async_def():
     (os.getcwd() + '/tests/fake_project/another_fake_module', True),
     (os.getcwd() + '/tests/fake_project/fake_module', True),
     (os.getcwd() + '/tests/fake_project/random_folder', False),
+    # Python 2.7
+    (os.getcwd() + '/tests/project_py27/another_fake_module', True),
+    (os.getcwd() + '/tests/project_py27/fake_module', True),
+    (os.getcwd() + '/tests/project_py27/random_folder', False),
```

Comment:
Would be nice to have the following folder structure:

```
tests/
   fake_project/
       py27/
       py3/
```

Seems a bit weird having `project_py27` and `fake_project` at the same level (at first glance isn't possible to say the difference between them).

Reply:
Ok, I'll do it :D